Neos.Fusion/Classes/FusionObjects/DataStructureImplementation.php

```diff
@@ -96,6 +98,21 @@ private function isUntypedProperty($property): bool
         if (!is_array($property)) {
             return false;
         }
-        return array_intersect_key(array_flip(Parser::$reservedParseTreeKeys), $property) === [];
+
+        $keysToIdentifyTypedProperty = array_flip(Parser::$reservedParseTreeKeys);
+        unset($keysToIdentifyTypedProperty['__meta']);
+
+        return array_intersect_key($keysToIdentifyTypedProperty, $property) === [];
+    }
+
+    /**
+     * Checks if the current instance (static) is really this very class (ignore proxy building)
+     * (fx. returns false, when this class was further inherited.)
+     *
+     */
+    private function isImplementationDataStructureImplementation(): bool
```

Comment:
1. The name should rather be `isImplementationADirectDataStructureImplementation()`
2. Is this still needed? The other mentioned related issues seem solved…